lib/ar_lazy_preload/active_record/association.rb

```diff
@@ -7,5 +7,18 @@ def load_target
       owner.try_preload_lazily(reflection.name)
       super
     end
+
+    def ids_reader # rubocop:disable Metrics/AbcSize
```

Comment:
Shouldn't `# rubocop:disable Metrics/AbcSize` be on top 
and re-enabled after the method?

Reply:
@PikachuEXE thanks, fixed it.